src/components/table/table.utils.js

```diff
@@ -0,0 +1,35 @@
+import { COMPONENT_TYPES } from "./table.constants";
+
+export function generateTableCell({
+  type = COMPONENT_TYPES.TABLE_CELL,
+  attributes = {},
+  text = "",
+} = {}) {
+  return {
+    type,
+    attributes,
+    text,
+  };
+}
+
+export function generateTableRow({ columns = [], attributes = {} }) {
+  const type = COMPONENT_TYPES.TABLE_ROW;
+  return { type, columns, attributes };
+}
+
+export function generateTable(
```

Comment:
Step 4: This is the factory method, which I used for mockTableData creation. Nothing fancy up here, it just takes in rows and columns and helps to build the table data which I can use it for the table and cell operation.